app/src/ui/dialog/dialog.tsx

```diff
@@ -180,6 +192,12 @@ export class Dialog extends React.Component<IDialogProps, IDialogState> {
       return
     }
 
+    if (this.props.maxWidth !== undefined) {
+      const widthSansPadding = window.innerWidth - 40
+      const newWidth = clamp(widthSansPadding, -infinity, this.props.maxWidth)
```

Comment:
I'm surprised this builds with the `infinity` Octicon 🤯 
```suggestion
      const newWidth = clamp(widthSansPadding, -Infinity, this.props.maxWidth)
```

Reply:
Lol. And it works..